src/shim.js

```diff
@@ -170,18 +185,8 @@ function executeCommand(command) {
     } else if (objectSpace[command.target]) {
         const target = objectSpace[command.target];
         const method = target[command.name];
-
-        if (haveCallbacks.indexOf(command.name) === -1) {
-            command.response = method.apply(target, command.params);
-            completeCommand(command);
-        } else {
-            let params = command.params.slice(); // copy params
-            params.push(status => {
-                command.response = status;
-                completeCommand(command);
-            });
-            method.apply(target, params);
-        }
+        command.response = method.apply(target, command.params);
+        completeCommand(command);
```

Comment:
Never mind, I realized `phantom.exit` still relies  on the `else if` statement. Do you think we should bind `phantom.exit` to `invokeFunction`, and deprecate the `else if`?